src/tiles/item-image.ts

```diff
@@ -114,6 +114,9 @@ export class ItemImage extends LitElement {
     if (this.model?.identifier) {
       gradientInt = this.generateHashCode(this.model.identifier) % 6;
```

Comment:
Shouldn't %6 always return an integer between 0 and 5 (inclusive). If so, this could just be
`gradientInt = (this.generateHashCode(this.model.identifier) % 6) + 1`